packages/nimble-components/src/table/components/row/template.ts

```diff
@@ -39,10 +70,13 @@ export const template = html<TableRow>`
                         column-id="${x => x.columnId}"
                         :recordId="${(_, c) => c.parent.recordId}"
                         ?has-action-menu="${x => !!x.actionMenuSlot}"
+                        ?expanded="${(_, c) => c.parent.expanded}"
+                        row-hierarchy-level="${(_, c) => c.parent.rowHierarchyLevel}"
+                        ?is-first-cell="${(_, c) => c.index === 0}"
                         action-menu-label="${x => x.actionMenuLabel}"
                         @cell-action-menu-beforetoggle="${(x, c) => c.parent.onCellActionMenuBeforeToggle(c.event as CustomEvent<MenuButtonToggleEventDetail>, x)}"
                         @cell-action-menu-toggle="${(x, c) => c.parent.onCellActionMenuToggle(c.event as CustomEvent<MenuButtonToggleEventDetail>, x)}"
-                        :nestingLevel="${(_, c) => c.parent.cellIndentLevels[c.index]};"
+                        :nestingLevel="${(_, c) => c.parent.cellIndentLevels[c.index]}"
```

Comment:
Can we use consistent terminology for "nesting level" vs "indent level"? Is there a meaningful distinction?

Reply:
Will revisit in follow-up work.